pkgs/desktops/gnome/core/gnome-color-manager/default.nix

```diff
@@ -28,6 +28,10 @@ stdenv.mkDerivation rec {
     sha256 = "1vpxa2zjz3lkq9ldjg0fl65db9s6b4kcs8nyaqfz3jygma7ifg3w";
   };
 
+  patches = [
+    ./exiv2-0.28.patch
```

Comment:
Could you please add a comment with a link to the patch source (e.g. upstream MR)?

Reply:
The patch I've made concerns the calibration part, which was removed in version 3.36.0 (whose update was blocked in #84734). How can it be upstreamed?

Reply:
If it is your patch, it might be a good idea to include the commit header such as the one produced by `git format-patch`  to make the authorship obvious.